R/html_dependencies.R

```diff
@@ -71,6 +71,19 @@ html_dependency_tocify <- function() {
     stylesheet = "jquery.tocify.css")
 }
 
+# Create an HTML dependency for pandoc_codeblock_accessibility
+#' @rdname html-dependencies
+#' @export
+html_dependency_accessibleCodeBlock <- function() {
+
+  htmlDependency(
+    name = "accessibleCodeBlock",
+    version = "0.0.1",
+    src = pkg_file("rmd/h/pandoc"),
```

Comment:
This will cause other js files be copied to `lib_dir` when `self_contained = FALSE`.
I recommend moving this file back to `accessibility`.
I guess you followed my suggestion, which was immediately removed after posting...
Sorry about this.